WatchDogIntelliJPlugin/WatchDog/src/nl/tudelft/watchdog/intellij/ui/wizards/userregistration/UserProjectRegistrationWizard.java

```diff
@@ -44,72 +44,27 @@ protected String getHelpID() {
     public void addSteps() {
 		userWelcomeStep = new UserWelcomeStep(0, this);
 		addStep(userWelcomeStep);
-		userRegistrationStep = new UserRegistrationStep(1, this);
-		addStep(userRegistrationStep);
-		existingUserEndingStep = new UserIdEnteredEndingStep(2, this);
-		addStep(existingUserEndingStep);
-		existingProjectIdStep = new ProjectIdEnteredEndingStep(3, this);
-		addStep(existingProjectIdStep);
-		projectWelcomeStep = new ProjectWelcomeStep(4, this);
-		addStep(projectWelcomeStep);
-		projectRegistrationStep = new ProjectRegistrationStep(5, this);
-		addStep(projectRegistrationStep);
-		projectSliderStep = new ProjectSliderStep(6, this);
-		addStep(projectSliderStep);
-		projectedCreatedStep = new ProjectCreatedEndingStep(7, this);
-		addStep(projectedCreatedStep);
-		this.totalSteps = 5;
+		projectCreatedStep = new ProjectCreatedEndingStep(1, this);
+		addStep(projectCreatedStep);
+		this.totalSteps = 2;
 	}
 
     @Override
 	public int getNextStep(int step) {
         if(this.getCurrentStepObject().canFinish()) return getCurrentStep();
-		if (myCurrentStep == userWelcomeStep.getStepId()
-				&& !userWelcomeStep.getRegisterNewId()) {
-			return existingUserEndingStep.getStepId();
-		}
-		if (myCurrentStep == existingUserEndingStep.getStepId()) {
-			return projectWelcomeStep.getStepId();
-		}
-		if (myCurrentStep == userRegistrationStep.getStepId()) {
-			return projectRegistrationStep.getStepId();
-		}
-		if (myCurrentStep == projectWelcomeStep.getStepId()
-				&& !projectWelcomeStep.getRegisterNewId()) {
-			return existingProjectIdStep.getStepId();
-		}
-		if (myCurrentStep == projectRegistrationStep.getStepId()
-				&& projectRegistrationStep.shouldSkipProjectSliderStep()) {
-			return projectedCreatedStep.getStepId();
-		}
-		if (myCurrentStep == projectSliderStep.getStepId()) {
-			return projectedCreatedStep.getStepId();
+		if (myCurrentStep == userWelcomeStep.getStepId()) {
+			return projectCreatedStep.getStepId();
 		}
 		return super.getNextStep(step);
 	}
 
     @Override
 	public int getPreviousStep(int step) {
         if(this.getCurrentStepObject().canFinish()) return -1;
```

Comment:
I think curly braces are missing and the formatting is wrong in the subsequent if (one step too indented)?